docs/getting-started/authenticating-to-github.md

```diff
@@ -1,15 +1,43 @@
+1. Click the **Connect** link in the GitHub section.
+        <a href="images/sign-in-to-github-provider.png?raw=true" target="_blank"><div><img src="images/sign-in-to-github-provider.png" alt="Connect to GitHub" height="300px"/></div></a>   
+
+   If you're connected to a TFS instance, click on the **Sign in** link instead
+   <a href="images/sign-in-to-github.png?raw=true" target="_blank"><div><img src="images/sign-in-to-github.png" alt="Sign in to GitHub" width="300px"/></div></a>  
+
+   If none of these options are visible, click **Manage Connections** and then **Connect to GitHub**.  
+   <a href="images/connect_to_github.png?raw=true" target="_blank"><div><img src="images/connect_to_github.png" alt="Connect to GitHub in the manage connections dropdown in the Team Explorer pane" width="500px"/></div></a>
+1. In the **Connect to GitHub dialog** choose **GitHub** or **GitHub Enterprise**, depending on which product you're using.
+
+**GitHub option**:
+<a href="images/connect-to-github-dialog.png?raw=true" target="_blank"><div><img src="images/connect-to-github-dialog.png" alt="Connect to GitHub dialog view" height="400px"/></div></a>
+
+- To sign in with credentials, enter either username or email and password.
+- To sign in with SSO, select `Sign in with your browser`.
+
+**GitHub Enterprise option**:  
+<a href="images/connect-to-github-enterprise-dialog.png?raw=true" target="_blank"><div><img src="images/connect-to-github-enterprise-dialog.png" alt="Connect to GitHub Enterprise dialog view" height="400px"/></div></a>
+
+- To sign in with SSO, enter the GitHub Enterprise server address and select `Sign in with your browser`.
+- To sign in with credentials, enter the GitHub Enterprise server address.
+   - If a `Password` field appears, enter your password.
+   - If a `Token` field appears, enter a valid token. You can create personal access tokens by [following the instructions in the section below](#personal_access_tokens).
 
 Before you authenticate, you must already have a GitHub or GitHub Enterprise account.
 
 - For more information on creating a GitHub account, see "[Signing up for a new GitHub account](https://help.github.com/articles/signing-up-for-a-new-github-account/)".
 - For a GitHub Enterprise account, contact your GitHub Enterprise site administrator.
 
-**Note:** If your organization is on the [Business plan](https://help.github.com/articles/organization-billing-plans) and has not enabled SAML single sign-on or login with username and password, you must create and authorize a personal access token to access protected content. In addition, SAML single sign-on is not available for GitHub enterprise versions less than 2.12.2.
+### Personal access tokens
 
-### Scopes for personal access tokens
+If all signin options above fail, you can manually create a personal access token and use it as your password.
 
 The scopes for the personal access token are: `user`, `repo`, `gist`, and `write:public_key`. 
```

Comment:
I think we should put this information right where the uses needs it.

At the moment we simply say `Token` where the user needs to enter their personal access token. Somehow they need to find this documentation or figure out what scopes are required by themselves.

![image](https://user-images.githubusercontent.com/11719160/39048003-e560a17e-4493-11e8-95e4-0a257f43dab7.png)

We could be much more helpful and change it to `Token (required scopes: `user`, `repo`, `gist`, and `write:public_key`).